packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -30,6 +41,8 @@ export interface DataListProps extends Omit<React.HTMLProps<HTMLUListElement>, '
   selectedDataListItemId?: string;
   /** Flag indicating if DataList should have compact styling */
   isCompact?: boolean;
+  /** Specifies the grid breakpoints  */
+  girdBreakpoint?: 'none' | 'always' | 'sm' | 'md' | 'lg' | 'xl' | '2xl';
```

Comment:
Not sure how I didn't catch this before... but `gird` should be `grid`.

Reply:
It's because I pushed the wrong string with the last update. thanks for catching this.